app/src/main/java/org/mozilla/fenix/settings/Extensions.kt

```diff
@@ -25,12 +26,12 @@ fun SitePermissions.toggle(featurePhone: PhoneFeature): SitePermissions {
 }
 
 /**
- * In devices with Android 6, when we use android:button="@null" android:drawableStart doesn't work via xml
+ * In devices with Android 6, when we use android:button="@null" app:drawableStartCompat doesn't work via xml
```

Comment:
Nit: did you check that this is still true with `drawableStartCompat`?  If not, I'd rather leave this comment as it was.  Ideally we would test the new version (and set this in the XML if it worked), but it's probably not worth the time to go digging for an appropriate device unless you have one handy.

Reply:
Ooops, you're right, I meant to go back and fix that.  Thank you for spotting it!